src/Components/Home.js

```diff
@@ -1,31 +1,152 @@
 import { Box, Typography } from "@mui/material";
 import React from "react";
+import { useState, useEffect } from "react";
+import cd1 from "../Assets/cd1.png";
+import cd2 from "../Assets/cd2.png";
+import cd3 from "../Assets/cd3.png";
+import S1 from "../Assets/S1.jpg";
+import p1 from "../Assets/p1.jpg";
+import p2 from "../Assets/p2.jpg";
+import CardSlider from "../Shared/CardSlider.js";
+import Testimonials from "../Shared/Testimonials.js";
+import { Swiper, SwiperSlide } from "swiper/react";
+import { Navigation, Pagination, Scrollbar, A11y } from "swiper/modules";
+import "swiper/css/navigation";
+import "swiper/css/pagination";
+import "swiper/css/scrollbar";
+import "swiper/css";
+import { EffectFade } from "swiper/modules";
 
+import Slider from "./Slider";
+import Content from "./Content";
 import "../Styles/Home.css";
+import SimpleCards from "../Shared/SimpleCards.js";
 const Home = () => {
+  const cardData = [
```

Comment:
variable name does not convey anything meaningful. Fix on all places @khadijatahira2001